goddard/schema.coffee

```diff
@@ -36,6 +36,15 @@ module.exports = exports = (app) ->
 	app.set('database', sequelize)
 
 	# set the models
+	Models.reports = sequelize.define('reports', {
+
+		nodeId: { type: Sequelize.INTEGER, field: 'nodeId' }
+		status: { type: Sequelize.STRING(255), field: 'status' }
+		message: { type: Sequelize.STRING(255), field: 'message' }
```

Comment:
Actually the timestamps from the file would be useless as if it fails you won't see that timestamp again so createdAt and updatedAt alone are better.